multi-int-store/src/main/java/com/connexta/multiintstore/services/api/RetrieveApi.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.multiintstore.services.api;
+
+import org.springframework.core.io.Resource;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+/** TODO Replace this with the real StoreAPI */
```

Comment:
Remove TODO comment.

Reply:
I put this here as a reminder to replace it with the separate API artifact that we will eventually create.